### ISSUE TITLE
No update since several month?

### ISSUE THREAD

**Issue by guettli** [vadikom/smartmenus#105]
What is the state of the project?

**Comment by vadikom** [comment]
Hi, first of all, I have no plans of shutting down the project. :)

But, of course, there are a few reasons behind the lack of notable recent activity:

- Nowadays jQuery is slowly being phased out so it doesn't make sense to invest much more time/efforts in the current code base considering all the major features I had in mind are already implemented and working well enough.
- Due to the above, I have been investing quite some time lately on planning and designing the next major release which will be a complete "pure JS" rewrite with a new repo structure, modular code base (e.g. the heaviest features that many users may not need would become addons/plugins, etc.), many new npm packages, etc. But that is still in quite early stages of development and I can't even give you a rough estimate when it would be available.
- The last release 1.1.0 has proven to be quite stable and no major issues have really been discovered so I have not been pushed too hard to rollout even a maintenance release - although there have been some "insignificant" (I mean not affecting many users) issues addressed since then.
- The time I am able to spend on development has sadly been getting less and less in the past few years due to a number of reasons (two small kids, client projects/work, even a bad football knee injury I had to take care of with a few surgeries and rehab). But that is really mainly going to affect the next major release.

So to sum it up, the current jQuery-dependent branch will most probably not see any new major feature releases but there will almost certainly be some maintenance releases and I do not plan to end support for it any time soon. Although jQuery is slowly losing popularity I am sure it will still be quite widely used in the following years.

**Comment by guettli** [comment]
Thank you very much for this super friendly profound answer! Write me a message if you need a beta tester for your new version. Take your time.

**Comment by guettli** [comment]
Just for the records, here is a pure JS based menu with accessibility support: https://github.com/NickDJM/accessible-menu

**Comment by Darkness7148** [comment]
Not really sure how you can say jquery is "being phased out". It's still by far the most popular js framework.

**Comment by vadikom** [comment]
I wrote: "<b>slowly</b> being phased out" :)

And also: "Although jQuery is slowly losing popularity I am sure it will still be quite widely used in the following years."

I don't mean to argue but there is really a lot of data supporting the case that jQuery is indeed slowly losing popularity (e.g. Bootstrap 5 won't require it). Built-in browser API's as well as coding practices (npm, modules, etc.) have evolved dramatically since jQuery was first introduced and thus nowadays less and less people rely on jQuery for new projects.

**Comment by venuscasinoinfo1** [comment]
Venus Casino - Thương hiệu Nhà cái hàng đầu, có trụ sở chính tại Bavet, Campuchia. Nhiều Khuyến mãi hấp dẫn cùng sản phẩm Game đa dạng như : Đá Gà, Slot, Casino, Bóng Đá, Bắn cá....Venuscasino.info là website chính thức và duy nhất của Venus Casino 
#venuscasino #venuscasinoinfo #venuscasinodotinfo
Website: https://venuscasino.info/
https://twitter.com/venuscasinoinf1
https://www.youtube.com/channel/UCN02I2NCn_1tevaTrrPlI6A/about
https://www.linkedin.com/in/venuscasinoinfo1/
https://dribbble.com/venuscasinoinfo11/about
https://www.plurk.com/venuscasinoinfo11
https://github.com/venuscasinoinfo1
https://www.pinterest.com/venuscasinoinfo11/_saved/
https://www.instagram.com/venuscasinoinfo11/
https://www.flickr.com/people/194508666@N04/
https://venuscasinoinfo11.tumblr.com/
